### DISCUSSION TITLE
ELF Relocations

### DISCUSSION THREAD

**Discussion by 0xc1f** [Vector35/binaryninja-api#2911] [Analysis] [answered]
ELF relocations are getting applied to the binary during analysis?

**Comment by 0xc1f** [comment]
I guess that I found relocation related issue on the newer BN version. I'd like to share the sample.

**Comment by fuzyll** [comment, marked as answer]
Hey, really sorry for the delay. If this is still an issue for you, are you able to point me to the issue (or create a new one) and either attach the sample (in a `.zip`) or email it to binaryninja@vector35.com (if you're willing to share, but it needs to be kept private)? That would be super helpful. Thanks!

**Comment by lwerdna** [comment]
issue: https://github.com/Vector35/arch-armv7/issues/61
